NEWS.md

```diff
@@ -1,3 +1,27 @@
+# bbr 1.8.0
+
+## New features and changes
+
+* New `inherit_param_estimates()` function for setting the initial parameter 
+  estimates of a model using the final estimates of a previously executed model. 
+  (#623)
+* Improved documentation and vignettes. (#622, #627, #631)
+* Some internal functions now rely on `nmrec` for parsing control stream files,
+  rather than regex/string manipulation. (#600, #603, #604, #606, #612)
```

Comment:
Maybe drop "internal" since with a quick glance at least one of the rewritten functions is exported?